python/foglamp/services/core/scheduler/scheduler.py

```diff
@@ -1078,6 +1077,50 @@ def _schedule_row_to_schedule(cls,
             self._schedule_first_task(schedule_row, now)
             self._resume_check_schedules()
 
+    async def ping_service(self, service):
```

Comment:
Don't we already have a way to ping a service?

Reply:
This is actually makes a call to the ping method of the individual service, via the defined route, which is part of the microservice instance.

Reply:
@amarendra-dianomic if its required, can it be a utils def; so that monitoring loop can also use it